src/pages/Lend/LoanChannelCategoryControl.vue

```diff
@@ -150,6 +150,8 @@
 				</div>
 			</div>
 		</div>
+
+		<kv-loading-overlay v-if="loading" />
```

Comment:
The loading states were not working properly on this page. This adds 1 loader, which displays inside the relative position wrapper div. Loader is visible, when this page loads without prefetch (linking from lbc experiment layout) or when changing pages. it shows for categories with the MG promo card and those without